pages/home/content.json

```diff
@@ -29,8 +29,19 @@
 	"theteamEmail": "email Sherlock",
 	"theteamEmailLink": "sherlock@bakerstreet.gov",
 	"theteamPhone": "0799786772",
-	"servicesH1": "Servies: The Brief",
-	"servicesCopy": "We'll work with you closely to fully understand your project requirements, from what you want to communicate to the style in which you'd like to do it. We'll then come up with a proposal, budget and time-frame that works for you.",
+	"servicesH1": "Services",
+	"services": [
```

Comment:
Let's create the objects we'd like to see the editor support:
```json
"services": [
  { title: "Brief", copy:"We'll work with you closely" }
]
```
I'd rather the text in here was all human readable... the current services look like they might be for robots, perhaps to pick an icon. I think we should slugify the human text rather than humanizing the robot text.